source/area_harvested_2014.csv

```diff
@@ -0,0 +1,9582 @@
+Domain Code,Domain,Area Code,Area,Element Code,Element,Item Code,Item,Year Code,Year,Unit,Value,Flag,Flag Description
```

Comment:
Ui, interesting. That is also a good source. Nice that it works out of the box with my script. :smiley: 

You also got this data from FAO, likely?